examples/README

```diff
@@ -2,11 +2,11 @@ This directory contains examples of client code that uses kafka.
 
 To run the demo:
 
-   1. Start Zookeeper and the Kafka server
+   1. Start Zookeeper and the Kafka server.Note, it's unnecessary to start Zookeeper in KRaft mode.
```

Comment:
I think this can still be improved like this:
`In Zk mode, Start Zookeeper and the Kafka server; in KRaft mode, start the Kafka server.`
Also note that we should have a space between a punctuation and a word.